Source/EasyTipView.swift

```diff
@@ -139,7 +139,6 @@ public extension EasyTipView {
             self.alpha = self.preferences.animating.dismissFinalAlpha
         }) { (finished) -> Void in
             completion?()
-            self.delegate?.easyTipViewDidDismiss(self)
```

Comment:
Yeah, though about it but it seems to me that it's more clear to call the delegate from the `dismiss()` completion rather then the animation completion block. 
However, no problem, we'll update the pull request.